packages/compartment-mapper/test/optional.test.js

```diff
@@ -41,7 +42,7 @@ scaffold(
     );
   },
   4,
-  { knownFailure: true },
+  { knownArchiveFailure: true },
```

Comment:
(not a polite way to say you should, just a note)
If you want to assert for both behaviors you could pass an assertion to scaffold that uses `testCategoryHint` to decide what to assert instead of this.

Reply:
Yeah, this is a sign things are getting weedy. We might refactor `knowFailure` to be a callback in the future.